### PR TITLE
Clean up amp-ad-exit experiment

### LINKED ISSUE #9502: Cleanup 'amp-ad-exit' experiment

### PR DESCRIPTION
**Pull request by clawr**
Clean up amp-ad-exit experiment. Fixes #9502. The experiment was launched with https://github.com/ampproject/amphtml/pull/10102.

### CODE REVIEW

File: build-system/global-configs/canary-config.json [hunk 1]
```diff
@@ -24,5 +24,4 @@
   "a4a-measure-get-ad-urls": 1,
   "ad-loader-v2": 1,
   "3p-use-ampcontext": 0,
-  "amp-ad-exit": 1
```
Comment (lannka) [inline build-system/global-configs/canary-config.json]: do the config change in a separate PR. otherwise it will fail our presubmit check

Comment (clawr) [inline build-system/global-configs/canary-config.json]: Done. https://github.com/ampproject/amphtml/pull/10430
